simtools/configuration/commandline_parser.py

```diff
@@ -231,26 +236,218 @@ def initialize_simulation_model_arguments(self, model_options):
+            required=True,
+            choices=[
+                "gamma",
+                "gamma_diffuse",
+                "electron",
+                "proton",
+                "muon",
+                "helium",
+                "nitrogen",
+                "silicon",
+                "iron",
+            ],
+        )
+        _configuration_group.add_argument(
+            "--azimuth_angle",
+            help=(
+                "Telescope pointing direction in azimuth. "
+                "It can be in degrees between 0 and 360 or one of north, south, east or west "
+                "North is 0 degrees and the azimuth grows clockwise (East is 90 degrees)."
+            ),
+            type=CommandLineParser.azimuth_angle,
+            required=True,
+        )
+        _configuration_group.add_argument(
+            "--zenith_angle",
+            help="Zenith angle in degrees (between 0 and 180).",
+            type=CommandLineParser.zenith_angle,
+            required=True,
+        )
+        _configuration_group.add_argument(
+            "--nshow",
+            help="Number of showers per run to simulate.",
+            type=int,
+            required=False,
+        )
+        _configuration_group.add_argument(
+            "--run_number_start",
+            help="Run number for the first run.",
+            type=int,
+            required=True,
+            default=1,
+        )
+        _configuration_group.add_argument(
+            "--number_of_runs",
+            help="Number of runs to be simulated.",
+            type=int,
+            required=True,
+            default=1,
+        )
+        _configuration_group.add_argument(
+            "--event_number_first_shower",
+            help="Event number of first shower",
+            type=int,
+            required=False,
+            default=1,
+        )
+
+    def _initialize_shower_configuration(self):
+        """Initialize shower configuration arguments."""
+        shower_config = self.add_argument_group("shower parameters")
+        shower_config.add_argument(
+            "--eslope",
+            help="Slope of the energy spectrum.",
+            type=float,
+            required=False,
+            default=-2.0,
+        )
+        shower_config.add_argument(
+            "--erange",
+            help="Energy range of the primary particle (min/max value, e'g', '10 GeV 5 TeV').",
+            type=CommandLineParser.parse_quantity_pair,
+            required=False,
+            default=["3 GeV 330 TeV"],
+        )
+        shower_config.add_argument(
+            "--viewcone",
+            help="Viewcone for primary arrival directions (min/max value, e.g. '0 deg 5 deg').",
+            type=CommandLineParser.parse_quantity_pair,
+            required=False,
+            default=["0 deg 0 deg"],
+        )
+        shower_config.add_argument(
+            "--core_scatter",
+            help="Scatter area for shower cores (number of use; scatter radius).",
```

Comment:
We pass the distance, not the area. So I would call it "X and Y distances that define the scatter area. Scatter radius would also be wrong, because the distance in each direction might be different.

Reply:
Correct, fixed.